src/game_display.hpp

```diff
@@ -172,8 +172,7 @@ class game_display : public display
 
 	std::set<map_location> units_that_can_reach_goal_;
 
-	std::vector<std::string> reach_map_images_;
-	std::vector<texture> get_reachmap_images(const map_location& loc);
+	const std::vector<texture> get_reachmap_images(const map_location& loc);
```

Comment:
Ah, I meant that the function itself should be const, not that it should return a const value. That is,
```cpp
std::vector<texture> get_reachmap_images(const map_location& loc) const;
```
sorry for the confusion!

Reply:
Don't worry, you were very concise, I just didn't quite understand. Should be addressed with d4fedc6

Reply:
Yes, that was what I thought, I learned a lot from this PR alone.